### PR TITLE
shorten stage message

### PR DESCRIPTION
**Pull request by darkonie**

### CODE REVIEW

**Comment by malnick** [comment]
@jfurrow Can you take a look?

File: ext/dcos-installer/dcos_installer/action_lib/__init__.py [hunk 1]
```diff
@@ -105,12 +105,12 @@ def _add_copy_packages(chain, local_pkg_base_path='/genconf/serve'):
 
         chain.add_execute(['mkdir', '-p', destination_package_dir], stage='Creating package dir')
         chain.add_copy(local_pkg_path, destination_package_dir,
-                       stage='Coping packages {}'.format(local_pkg_path))
+                       stage='Coping packages')
```
Comment (jfurrow) [inline ext/dcos-installer/dcos_installer/action_lib/__init__.py]: This should be `Copying`

**Comment by jfurrow** [comment]
One minor comment, otherwise this looks good! Thanks for doing that!

File: ext/dcos-installer/dcos_installer/action_lib/__init__.py [hunk 2]
```diff
@@ -105,12 +105,12 @@ def _add_copy_packages(chain, local_pkg_base_path='/genconf/serve'):
...
 
 
 def _add_copy_bootstap(chain, local_bs_path):
     remote_bs_path = REMOTE_TEMP_DIR + '/bootstrap'
-    chain.add_execute(['mkdir', '-p', remote_bs_path], stage='Creating dir {}'.format(remote_bs_path))
+    chain.add_execute(['mkdir', '-p', remote_bs_path], stage='Creating dir')
```
Comment (jfurrow) [inline ext/dcos-installer/dcos_installer/action_lib/__init__.py]: Perhaps we could spell out `directory` to be a little more human-friendly?

**Comment by darkonie** [comment]
@malnick @jfurrow updated, please check one more time

**Comment by jfurrow** [comment]
LGTM :ship:

**Comment by cmaloney** [comment]
:ship:

**Pull request merged**